test/ceil.test.js

```diff
@@ -0,0 +1,27 @@
+// Importing require so it works on Node.js >14
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+
+// Chai assert
+const assert = require('chai').assert;
+
+import ceil from "../src/ceil.js";
+
+/****** 6. ceil.js tests *****/
+describe("ceil.js tests", function() {
```

Comment:
You should add type -tests for all the files which have specified a certain return type. For ceil.js it is number.